dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GA4GHIT.java

```diff
@@ -45,9 +54,10 @@
  */
 public abstract class GA4GHIT {
     protected static final DropwizardTestSupport<DockstoreWebserviceConfiguration> SUPPORT = new DropwizardTestSupport<>(
-        DockstoreWebserviceApplication.class, CommonTestUtilities.CONFIDENTIAL_CONFIG_PATH);
+        DockstoreWebserviceApplication.class, CommonTestUtilities.PUBLIC_CONFIG_PATH, ConfigOverride.config("database.properties.hibernate.hbm2ddl.auto", "validate"));
```

Comment:
Why is this changing? 
(may be worth a code comment)

Reply:
GA4GHIT never needed confidential info, but did need some data.  Could keep it as confidential yaml but override basePath as "/api/", that would work.  But changing confidential yaml to have "/api" would break a bunch of unrelated tests.